node/src/commands/deploy.js

```diff
@@ -36,8 +36,8 @@ const deploy = async (options, variables) => {
     deployment = environment.latestDeploymentLog;
   } else {
     assertNoWorkspaceNameChanges(options, environment);
-    const optionsWithoutWorkspace = { ...options }
-    delete optionsWithoutWorkspace[WORKSPACE_NAME]
+    const optionsWithoutWorkspace = { ...options };
+    delete optionsWithoutWorkspace[WORKSPACE_NAME];
```

Comment:
there's no prettier/linter I presume? if so , should we open a ticket for next PRs?

Reply:
Yes

Reply:
Created: 
[Add Husky and Github lint checker to the CLI repo on env0 - RnD | Trello](https://trello.com/c/ZyEK7h8q/5718-add-husky-and-github-lint-checker-to-the-cli-repo) https://trello.com/c/ZyEK7h8q/5718-add-husky-and-github-lint-checker-to-the-cli-repo